lectures/12-plonk.tex

```diff
@@ -0,0 +1,234 @@
+\documentclass[../lecture-notes.tex]{subfiles}
+\usepackage[T1]{fontenc}
+\usepackage{lmodern}
+\usepackage{xcolor}
+\usepackage{fix-cm}
+\pgfplotsset{compat=1.18}
+
+\begin{document}
+
+\subsection{Preliminaries}
+
+Before delving into Plonk, let's first consider a few useful proving system gadgets.
+
+Let $\Omega$ be some subset of $\mathbb{F}_p$ of size $k$.
```

Comment:
You can reference Section 3.1.3 for more details. Also, I believe this is not an arbitrary subset, but rather a multiplicative cyclic subgroup of $\mathbb{F}_p^{\times}$?